test/crosschain/trade/flyTrade.test.ts

```
@@ -0,0 +1,86 @@
+        })
+
+        test('returns 0 when value not found', () => {
+            const callData = '0x' + 'ff'.repeat(64)
+            expect(FlyTrade.findValueOffset(callData, '42')).toBe(0)
+        })
+
+        test('found offset is patchable round-trip', () => {
+            const fromAmount = BigNumber.from('999888777666')
+            const padding = '0'.repeat(128)
+            const callData =
+                '0xdeadbeef' + padding + fromAmount.toHexString().slice(2).padStart(64, '0') + padding
```

ESLint

[error] Delete `⏎···············`

(prettier/prettier)